### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by abellizzistudent**
@gosusleeper  
Updated POST method and it now works. This is required to move forward with Assignment 3.

### CODE REVIEW

File: cst438_register/src/main/resources/application.properties [hunk 1]
```diff
@@ -9,15 +9,16 @@ registration.url = http://localhost:8080
 registration.service = default
 
 # database properties
-# spring.datasource.url = jdbc:mysql://localhost:3306/register
-# spring.datasource.username = root
-# spring.datasource.password = dw4$Database
+ spring.datasource.url = jdbc:mysql://localhost:3306/register
+ spring.datasource.username = root
+ spring.datasource.password = root
```
Comment (gosusleeper) [inline cst438_register/src/main/resources/application.properties:14]: You accidentally pushed properties. It shows the password. Be careful not to commit properties if it isn't necessary.

Comment (gosusleeper) [reply cst438_register/src/main/resources/application.properties:14]: It is local so no harm no foul.

File: cst438_register/src/main/java/com/cst438/controller/StudentController.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+			ns = studentRepository.save(ns); //update ns obj with DB key
+			student.id = ns.getStudent_id(); //
+			return student;
+		} else {
+			throw  new ResponseStatusException( HttpStatus.BAD_REQUEST, "Duplicate email in use.  " + student.studentEmail);
+		}
+	}
+	
+//As an administrator, I can put student registration on HOLD
+
```
Comment (gosusleeper) [inline cst438_register/src/main/java/com/cst438/controller/StudentController.java:50]: I like the listing of the user stories.

File: cst438_register/src/main/java/com/cst438/domain/StudentDTO.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
+package com.cst438.domain;
+
```
Comment (gosusleeper) [inline cst438_register/src/main/java/com/cst438/domain/StudentDTO.java:2]: I like how you coded a StudentDTO.java and StudentController.java

Comment (gosusleeper) [reply cst438_register/src/main/java/com/cst438/domain/StudentDTO.java:2]: Your new code is very clean. Giving a separate file is a fresh start and makes the review a lot more readable.

File: cst438_register/src/main/java/com/cst438/controller/StudentController.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+	@Autowired
+	StudentRepository studentRepository;
+
+//For development testing, variables here will need to be removed
+	//String email ="test@test.com";
+	//String name ="First Last";
+	
+	@PostMapping("/student")
+	@Transactional	
+//Method that is called when POST request is made
```
Comment (gosusleeper) [inline cst438_register/src/main/java/com/cst438/controller/StudentController.java:32]: Code is well documented and easy to follow along.

File: cst438_register/src/main/resources/application.properties
Comment (gosusleeper) [inline cst438_register/src/main/resources/application.properties:1]: Testing commenting on the file.

File: cst438_register/src/main/resources/application.properties (hunk 1 above)
Comment (abellizzistudent) [reply cst438_register/src/main/resources/application.properties:14]: Good catch, thanks I'll fix that up.

**Pull request merged**